### ISSUE TITLE
Allow vuex to be used in component

### ISSUE THREAD

**Issue by mpjraaij** [richardeschloss/nuxt-socket-io#88]
Hi Richard,

I'm nothing that right now I can only use the vuex option in the nuxt.config file.
Is it possible to put this in the vue component instead?

**Comment by richardeschloss** [comment]
Hi, that's what #86 is on the roadmap to solve. The idea will be that vuexOpts can also be passed in to nuxtSocket instance options, to override options in nuxt.config. I'll close this issue now and will happily work on this. I have another important feature I just merged in that I need to document while the info is still fresh in my head.

**Comment by richardeschloss** [comment]
@mpjraaij feature implemented in v1.0.23, just released now at midnight. "vuex" can now be passed in component:

```
mounted() {
  this.socket = this.$nuxtSocket({
    ...,
    vuex: { 
      mutations: [],
      actions: ['],
      emitBacks: []
    }
  })
}
```

Friendly reminder: mutations and actions set up *listeners* on the socket, so it will be best practice to avoid duplicates (the plugin does try it's best to not allow duplicates). I hope this helps!
